### PR TITLE
[FEAT] Added a Map Modal using Google's Map API

### PR DESCRIPTION
**Pull request by PJhaveri02**
Added a map modal using Google's map API so users can see a restaurant's location on a map.

### CODE REVIEW

**Review by nbed807 — DISMISSED** [review]

File: frontend/src/components/8.Result/ResultPage.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import '../Common/Help.css';
 import MapModal from '../Common/MapModal';
 import Icon from '../Common/MapsPinpoint';
 
-const mapSrc = 'https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d3192.588180029826!2d174.7669186152492!3d-36.85233777993783!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x6d0d47e383f32253%3A0xbd49f61f758a9e5b!2sThe%20University%20of%20Auckland!5e0!3m2!1sen!2snz!4v1615862553109!5m2!1sen!2snz';
+// const mapSrc = 'https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d3192.588180029826!2d174.7669186152492!3d-36.85233777993783!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x6d0d47e383f32253%3A0xbd49f61f758a9e5b!2sThe%20University%20of%20Auckland!5e0!3m2!1sen!2snz!4v1615862553109!5m2!1sen!2snz';
```
Comment (jemimahmartinez) [inline frontend/src/components/8.Result/ResultPage.js:8]: We could get rid of this now since we're not using it

**Review by jemimahmartinez — CHANGES_REQUESTED** [review]
Just left one nitpicky comment. Otherwise, it's looking good 👌

**Review by nbed807 — COMMENTED** [review]
LGTM 🙂

**Review by jemimahmartinez — APPROVED** [review]
Awesome work 👍 LGTM

**Review by nbed807 — APPROVED** [review]
That all looks great!

**Review by Alxjlv — APPROVED** [review]
👍

**Review by aniket-rai — APPROVED** [review]
LGTM. Good work man!

**Pull request merged**